### PR TITLE
ALL-7315 - Add Cosmos grpc

### PR DESCRIPTION
**Pull request by Hathoriel**
# Add Cosmos GRPC

### CODE REVIEW

File: src/dto/rpc/CosmosRpcSuite.ts [hunk 1]
```diff
@@ -0,0 +1,1600 @@
...
+  block_height: string;
+  validators: Validator[];
+  pagination: PaginationResponse;
+}
+
+export interface GetValidatorSetByHeightRequest extends PaginationRequest {
+  height: number;
+}
+
+// Get validator set by height response interface
```
Comment (alexloiko) [inline src/dto/rpc/CosmosRpcSuite.ts:353]: do we really need all these comments? looks like they are just duplicating interfaces names

Comment (Hathoriel) [reply src/dto/rpc/CosmosRpcSuite.ts:353]: yes will remove this.

Comment (Hathoriel) [reply src/dto/rpc/CosmosRpcSuite.ts:353]: Next time I will use https://github.com/drwpow/openapi-typescript or something like this

Comment (Hathoriel) [reply src/dto/rpc/CosmosRpcSuite.ts:353]: Or this OpenApi3-TS

**Review by alexloiko — APPROVED** [review]

**Pull request merged**